external-api/src/http/admin.rs

```diff
@@ -11,6 +11,12 @@ use serde::{Deserialize, Serialize};
 pub const IS_LEADER_ROUTE: &str = "/v0/admin/is-leader";
 /// Get the open orders managed by the node
 pub const ADMIN_OPEN_ORDERS_ROUTE: &str = "/v0/admin/open-orders";
+/// Route to create a matching pool
+pub const ADMIN_MATCHING_POOL_CREATE_ROUTE: &str = "/v0/admin/:matching_pool/create";
+/// Route to destroy a matching pool
+pub const ADMIN_MATCHING_POOL_DESTROY_ROUTE: &str = "/v0/admin/:matching_pool/destroy";
+/// Route to create an order in a matching pool
+pub const ADMIN_CREATE_ORDER_ROUTE: &str = "/v0/admin/wallet/:wallet_id/orders/:matching_pool";
```

Comment:
nit: maybe just `/v0/admin/wallet/:wallet_id/order-in-pool` and take the `matching-pool` as a parameter of the request body. Again, the thinking here is that we're modifying a scoped object of the wallet, which a matching pool is not, but an order is